webapp/src/Service/CheckConfigService.php

```diff
@@ -337,6 +346,36 @@ public function checkTmpdirWritable()
                  $tmpdir)];
     }
 
+    public function checkHashTime()
+    {
+        $tmpuser = new User();
+        $counter = 0;
+        $time_start = microtime(true);
+        do {
+            $plainPassword = 'ryanpass';
+            $encoded = $this->passwordEncoder->encodePassword($tmpuser, $plainPassword);
+            // $user->setPassword($encoded);
+            $time_end = microtime(true);
+            $counter++;
+        } while ( ($time_end - $time_start) < 4);
+
+        $length_suggestion = 'When you have a contest duration more than a week increase the password length.';
+        if ($counter>300) {
+            return ['caption' => 'User password hashing',
+                'result' => 'W',
+                'desc' => sprintf('Hashing is to simple for small sized contests (Did %d hashes). %s',
```

Comment:
`s/to/too`
Also maybe we need to add somewhere in the docs how to increase / lower the hashing cost?

Reply:
Done